specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/preview/2020-07-01-preview/examples/AzureADMetricsUpdate.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "rg1",
+    "azureADMetricsName": "ddb1",
+    "api-version": "2020-07-01-preview",
+    "azureADMetricsConfig": {}
```

Comment:
>{} [](start = 28, length = 2)

please supply some tags here for completeness and propagate them to the response